src/Module.Server/Common/CrpgServerConfiguration.cs

```diff
@@ -25,11 +25,28 @@ public static void Init()
     public static CrpgRegion Region { get; }
     public static string Service { get; }
     public static string Instance { get; }
+    public static float ClanGroupSizePenalty { get; private set; } = 0f;

```

Comment:
Since it's a global variable it's important to give as much context as possible in the name. We often say
> The length of the identifier should be proportional to the distance between its declaration and use.

So here you could go with `TeamBalancerClanGroupSizePenalty` and `crpg_team_balancer_clan_group_size_penalty`

Reply:
done